dashboard/src/main/home/infrastructure/components/DeployList.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { useContext, useEffect, useState } from "react";
+import { Context } from "shared/Context";
+import api from "shared/api";
+import styled from "styled-components";
+import Loading from "components/Loading";
+import { Operation, OperationStatus, OperationType } from "shared/types";
+import { readableDate } from "shared/string_utils";
+import Placeholder from "components/Placeholder";
+
+type Props = {
+  infra_id: number;
+};
+
+const DeployList: React.FunctionComponent<Props> = ({ infra_id }) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+  const [operationList, setOperationList] = useState<Operation[]>([]);
+  const { currentProject, setCurrentError } = useContext(Context);
+
+  useEffect(() => {
+    api
+      .listOperations(
+        "<token>",
+        {},
+        {
+          project_id: currentProject.id,
+          infra_id: infra_id,
+        }
+      )
+      .then(({ data }) => {
+        if (!Array.isArray(data)) {
+          throw Error("Data is not an array");
+        }
+
+        setOperationList(data);
+        setIsLoading(false);
+      })
+      .catch((err) => {
+        console.error(err);
+        setHasError(true);
+        setCurrentError(err.response?.data?.error);
+        setIsLoading(false);
+      });
+  }, [currentProject]);
```

Comment:
Remember adding the infra_id to refresh the operations when it changes 

```suggestion
  }, [currentProject, infra_id]);
```